wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -1,78 +1,202 @@
+- **Avoid using the same hitsound for multiple notes in a chord.** This causes feedback to be louder than necessary. This also applies to active hitsounds and storyboard hitsounds used at the same timestamp.
+- **Timing changes should have a unified scroll speed by adjusting slider velocity.** This is done by normalizing the scroll speed of all timing sections to the most used one in the beatmap. The number is shown in brackets after the BPM range in the song selection, e.g. BPM: 60-160 (130). Note that this guideline does not affect the ability to add slider velocity changes which are used to create SV effects.
+- **Slider velocity changes should follow the general intensity of the song.**
+- **Slider velocity changes should be snapped according to the song's beat structure.** If the song is using a straight beat, 1/4, 1/8, 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used. If the song has a beat in a different position from what is recommended, snapping to an actual beat always takes priority.
+- **Avoid spamming short kiai time segments.** Doing so can be distracting and can negatively affect gameplay.
+- **Breaks should only be used when a section of music is extremely calm or silent.** Lower difficulties, especially on higher BPM songs, may use breaks to avoid parts of a song that are unreasonable for new players to play.
+
+## Difficulty-specific
+
+Difficulty-specific rules and guidelines do only apply to the difficulty level they are listed for and therefore do not apply to every *osu!mania* difficulty. Rhythm related guidelines apply to approximately 180 BPM maps. If your song is drastically faster or slower, some variables might be different. Apply reasonable judgement in these cases.
+
+*Key mode-specific guidelines are constructed based on the most common Ranked key modes (4 key and 7 key). Any other key modes should apply these guidelines when applicable*
+
+### Easy
+
+#### Rules
+
+- **Slider velocity changes that alter the scrolling speed of the beatmap are disallowed.** Slider velocity changes to unify the scroll speed in variable BPM beatmaps are exempt.
+- **Scroll normalization must be done in beatmaps with variable BPM.**
+- **A long note cannot be released during another long note’s body.** This requires a level of finger independence that players of this difficulty cannot handle.
+
+#### Guidelines
+
+- **Note snappings of consecutive 1/4 and higher should not be used.** Slower rhythms are more appropriate for this difficulty level.
+- **Long notes should be held for at least one beat.** Faster release timing is too difficult for players of this level.
+- **Objects should not be placed during a 1/1 long note’s hold duration.** However, using an object at the beginning or end of a long note is fine.
+- **HP and OD values should not be higher than 7.**
```

Comment:
imo this would look neater if HP and OD recommendations were the first guideline in all sections for this stuff